From 5b49ff99e1d5fb643a0a1aab88f771ecbd93678d Mon Sep 17 00:00:00 2001
From: Zhenyu Wang <zhenyu.z.wang@intel.com>
Date: Thu, 9 Oct 2008 09:19:41 +0800
Subject: [PATCH 14/20] Remove Lenovo T61 TV quirk

BIOS already says no integrated TV, and this quirk blocks TV
on R61i which has the same subdevice id as T61.
(cherry picked from commit 4128b01ec84ffa0f03c335c36738f383376d9381)
---
 src/i830_quirks.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git xf86-video-intel-2.4.3/src/i830_quirks.c xf86-video-intel-2.4.3/src/i830_quirks.c
index 35a891d..eb8bd82 100644
--- xf86-video-intel-2.4.3/src/i830_quirks.c
+++ xf86-video-intel-2.4.3/src/i830_quirks.c
@@ -286,8 +286,6 @@ static i830_quirk i830_quirk_list[] = {
 
     /* Lenovo Napa TV (use dmi)*/
     { PCI_CHIP_I945_GM, 0x17aa, SUBSYS_ANY, quirk_lenovo_tv_dmi },
-    /* Lenovo T61 has no TV output */
-    { PCI_CHIP_I965_GM, 0x17aa, 0x20b5, quirk_ignore_tv },
     /* Lenovo 3000 v200 */
     { PCI_CHIP_I965_GM, 0x17aa, 0x3c18, quirk_ignore_tv },
 
-- 
1.6.2

